src/core/main.c

```diff
@@ -2032,6 +2032,14 @@ static int invoke_main_loop(
         assert(ret_switch_root_init);
         assert(ret_error_message);
 
+        log_warning("sd: i=userspace, inactive_exit="USEC_FMT, m->timestamps[MANAGER_TIMESTAMP_USERSPACE].monotonic);
```

Comment:
If anything, these definitely shouldn't be logged at the warning level (debug seems to be appropriate).

Reply:
and they can all be queried at runtime, and some are already logged later, so doesn't seem worth adding any of these